open-apparel-registry/open-apparel-registry
# Remove most production resources

## Overview

we're updating the terraform to remove all resources from the OAR production environment, except for cloudfront and its related DNS and certificate resources.

Connects #2376

## Demo
attached is the output from running `scripts/infra plan` in this branch
[2376_plan_output.txt](https://github.com/open-apparel-registry/open-apparel-registry/files/10885656/2376_plan_output.txt)

## Notes

## Testing Instructions

* Run a `scripts/infra plan` in this branch and review the output to verify that nothing necessary is being removed

## Checklist
